frontend/src/components/pages/VerifyEmail.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useContext, useEffect } from "react";
+import { Flex, Image, VStack, Center, Text } from "@chakra-ui/react";
+import { useQuery } from "@apollo/client";
+import { Redirect } from "react-router-dom";
+import RHSLogo from "../../assets/RHSlogo.png";
+import AuthContext from "../../contexts/AuthContext";
+import { GET_USER_WITH_VERIFICATION_STATUS_BY_EMAIL } from "../../APIClients/queries/UserQueries";
+import { MANAGE_COURSES_PAGE } from "../../constants/Routes";
+import { AuthUser } from "../../types/AuthTypes";
+import authAPIClient from "../../APIClients/AuthAPIClient";
+
+const VerifyEmail = (): React.ReactElement => {
+  const { authUser, setAuthUser } = useContext(AuthContext);
+  const { refetch } = useQuery<
+    { userWithVerificationStatusByEmail: Omit<AuthUser, "accessToken"> },
+    { email: string }
+  >(GET_USER_WITH_VERIFICATION_STATUS_BY_EMAIL, { skip: true });
+  useEffect(() => {
+    if (authUser && authUser.email) {
+      authAPIClient.getAuthUser(authUser, refetch).then((user) => {
+        setAuthUser(user);
+      });
+    }
+  }, [authUser, setAuthUser, refetch]);
+
+  if (authUser && authUser?.emailVerified) {
+    return <Redirect to={MANAGE_COURSES_PAGE} />;
+  }
+
+  return (
+    <Flex minH="100vh">
+      <Center flex="1">
+        <VStack>
+          <Image height="13vh" marginBottom="2.5vh" src={RHSLogo} />
+          <Text variant="display-md" paddingBottom="1vw">
+            Please Verify your Email to Continue
+          </Text>
+        </VStack>
+      </Center>
+    </Flex>
```

Comment:
Optional: I'm assuming this is identical to the code on the login page. What does the feasibility of refactoring this to a reusable wrapper component look like? e.g.
```suggestion
  <AuthPage>
    <Text variant="display-md" paddingBottom="1vw">
      Please Verify your Email to Continue
    </Text>
  </AuthPage>
```

Reply:
We definitely can, but I don't know if AuthPage is the right name for it tbh. It's good to have it for the sake of consistency even if it's not much.

Reply:
Yeah, AuthPage was just me spitballing. I'll make a separate ticket for this.